python/seldon_core/env_utils.py

```diff
@@ -0,0 +1,35 @@
+"""
+Utilities to deal with Environment variables
+"""
+import json
+import os
+
+ENV_MODEL_NAME = "PREDICTIVE_UNIT_ID"
+ENV_MODEL_IMAGE = "PREDICTIVE_UNIT_IMAGE"
+ENV_SELDON_DEPLOYMENT_NAME = "SELDON_DEPLOYMENT_ID"
+ENV_PREDICTOR_NAME = "PREDICTOR_ID"
+ENV_PREDICTOR_LABELS = "PREDICTOR_LABELS"
+NONIMPLEMENTED_MSG = "NOT_IMPLEMENTED"
+NONIMPLEMENTED_IMAGE_MSG = f"{NONIMPLEMENTED_MSG}:{NONIMPLEMENTED_MSG}"
+
+
+def get_predictior_version(default_str: str = NONIMPLEMENTED_MSG) -> str:
```

Comment:
typo: `get_predictor_version` instead of `get_predictior_version`

Reply:
nice catch